tests/zeus/api/resources/test_repository_builds.py

```diff
@@ -105,9 +105,9 @@ def test_repo_build_create_multiple_authors(
     other_author = factories.AuthorFactory(
         repository=default_repo, email="bizzy@example.com"
     )
-    default_revision.authors.append(default_author)
     default_revision.authors.append(other_author)
     db_session.add(default_revision)
+    db_session.flush()
```

Comment:
this caused an integrity error in an odd code path given the changes werent flushed *before* executing the actual test logic